herbert_core/admin/is_jenkins.m

```diff
@@ -1,22 +1,25 @@
-function [is_jenkins_pc,job_name] = is_jenkins()
+function [is_jenkins_pc,job_name,workspace] = is_jenkins()
 % Attempt to determine if the PC we're running on is a Jenkins server.
 %   This is detected through environment variables that are always set by
 %   Jenkins.
 %
 % Returns:
 % is_jenkins -- true if the progam is running on Jenkins
-% job_name   -- the value of the Jenkins variable "JOB_NAME", 
+% job_name   -- the value of the Jenkins variable "JOB_NAME",
 jenkins_env_vars = {'JENKINS_URL', 'JOB_URL', 'JENKINS_HOME'};
-job_name ='';
-
+job_name = '';
+workspace = '';
 for i = 1:length(jenkins_env_vars)
     if isempty(getenv(jenkins_env_vars{i}))
         is_jenkins_pc = false;
         return;
     end
 end
 is_jenkins_pc = true;
+workspace = getenv('WORKSPACE');
```

Comment:
for consistency with the JOB_NAME test below should this be wrapped if an `if nargout > 2` ?